scripts/cbmc_json_parser.py

```diff
@@ -325,11 +356,18 @@ def postprocess_results(properties, extra_ptr_check):
     of the special cases above was hit.
     """
 
-    has_reachable_unsupported_constructs = has_check_failure(properties, GlobalMessages.UNSUPPORTED_CONSTRUCT_DESC)
-    has_failed_unwinding_asserts = has_check_failure(properties, GlobalMessages.UNWINDING_ASSERT_DESC)
+    has_reachable_unsupported_constructs = has_check_failure_message(
+        properties, GlobalMessages.UNSUPPORTED_CONSTRUCT_DESC)
+    has_failed_unwinding_asserts = has_check_failure_message(properties, GlobalMessages.UNWINDING_ASSERT_DESC)
     has_reachable_undefined_functions = modify_undefined_function_checks(properties)
+    has_failed_sanity_check = has_check_failure_property_class(properties, GlobalMessages.SANITY_CHECK)
```

Comment:
I think sanity checks should be handled differently: if one fails, we should print the one that failed and not print any results at all, i.e. error out. This should be done in a separate PR.

Reply:
I can address that right away in another PR.

Reply:
Can we take out sanity checks from this PR? Sanity checks are different than unwinding asserts and undefined functions in that if one fails, none of the results (whether SUCCESS or FAILURE) can be trusted.